src/components/modals/EditMembersModal.tsx

```diff
@@ -287,6 +273,8 @@ function Item({
 
     const owner = useWalletAccount()
 
+    const displayName = useENSName(address) || trunc(address)
```

Comment:
Normally we'd use `useDisplayUsername` but we do operate on `alias` here so we need the "non-alias" display name (ENS domain or raw 0x address).